Hearthstone Deck Tracker/Config.cs

```diff
@@ -753,6 +753,9 @@ public class Config
 		[DefaultValue(false)]
 		public bool ShowCapturableOverlay = false;
 
+		[DefaultValue(ShowDateOnDecksOptions.showNoDate)]
+		public ShowDateOnDecksOptions ShowDateOnDecksOptionsActive = ShowDateOnDecksOptions.showNoDate;
```

Comment:
The name is very unclear. Should probably be something like `SelectedDateOnDecks`.